README.md

```diff
@@ -1,47 +1,92 @@
 ---
 page_type: sample
 languages:
-- csharp
+  - csharp
 products:
-- azure
-extensions:
-- services: Service-Bus
-- platforms: dotnet
+  - azure
+  - dotnet
+  - azure-service-bus
+description: "This sample shows how to manage Service Bus manage Queues with claims based authorization"
+urlFragment: service-bus-dotnet-manage-with-claims-based-authorization
 ---
 
-# Getting started on managing Service Bus with claims based authorization in C# #
+# Getting started on managing Service Bus with claims based authorization in C#
```

Comment:
I don't know that this title is accurate;  with this title, I would expect to be seeing something that demonstrates a management plane library doing ARM functions .   As currently structured, this sample isn't focused on that.

Reply:
After reconsider this sample, I think this is not too bad to use at here.